### PR TITLE
chore(lint): turn on the 'import/no-duplicates' rule

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
This prohibits doing things like:

```ts
import { Pie } from './dessert';
import { Cake } from './dessert';
```

in favor of

```ts
import { Pie, Cake } from './dessert';
```

See https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/no-duplicates.md for details.

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Unit tests (`npm test`) were run locally and passed
- [ ] E2E Tests (`npm run test.karma.prod`) were run locally and passed
- [ ] Prettier (`npm run prettier`) was run locally and passed

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): linting!

## What is the current behavior?

GitHub Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by tanner-reits — APPROVED** [review]
I love me some good lint rules

File: package-lock.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@stencil/core",
   "version": "3.2.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```
Comment (rwaskiewicz) [inline package-lock.json:4]: Ah, good old NPM v9 bumped the lockfile version for us - that explains the number of lines changed 🙃

File: .eslintrc.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   root: true,
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint', 'jsdoc', 'jest', 'simple-import-sort'],
+  plugins: ['@typescript-eslint', 'jsdoc', 'jest', 'simple-import-sort', 'import'],
```
Comment (rwaskiewicz) [inline .eslintrc.js:4]: Nit - can we alphabetically sort these? (Assuming the order `import` is applied doesn't matter with relation to the others).

File: .eslintrc.js [hunk 3]
```diff
@@ -79,6 +79,7 @@ module.exports = {
     'prefer-spread': 'error',
     'simple-import-sort/imports': 'error',
     'simple-import-sort/exports': 'error',
+    'import/no-duplicates': 'error',
```
Comment (rwaskiewicz) [inline .eslintrc.js:82]: Nit - can we alphabetically sort these

**Review by rwaskiewicz — APPROVED** [review]
Two nitpicks, but non-blocking

File: .eslintrc.js (hunk 2 above)
Comment (alicewriteswrongs) [reply .eslintrc.js:4]: I don't think it should matter, especially since we're only turning on the one rule

File: .eslintrc.js (hunk 3 above)
Comment (alicewriteswrongs) [reply .eslintrc.js:82]: 🅰️🅱️🌊

**Pull request merged**